docs/user_guides/fs/feature_view/feature-server.md

```diff
@@ -0,0 +1,275 @@
+# Feature Store REST API Server
```

Comment:
Please add the description metadata to this page.

Reply:
added

Reply:
No you need to add a space between `description:` and `Using`